docs/documentation/features/internationalization.md

```diff
@@ -132,6 +132,182 @@ channel-type.yahooweather.temperature.option.OPTION2 = Option Nummer 2
 
 So the key for referencing a label of a defined thing type is `thing-type.<binding-id>.<thing-type-id>.label`. A label of a channel can be referenced with `channel-type.<binding-id>.<channel-type-id>.label`. And finally the config description parameter key is `thing-type.config.<binding-id>.<thing-type-id>.<parameter-name>.label` and the group parameter is `thing-type.config.<binding-id>.group.<thing-type-id>.<parameter-name>.label`.
 
+The following snippet shows an excerpt of the thing type definition XML file of the Weather Underground Binding and its language file that localizes labels and descriptions for the French language.
+
+XML file (thing-types.xml):
+
+```xml
+<thing:thing-descriptions bindingId="weatherunderground" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Remove the XML schema definitions. They are never used in this documentation page.